manifests/install.pp

```diff
@@ -1,13 +1,24 @@
 class solr5::install inherits solr5 {
-  download_uncompress { $package_url:
-    distribution_name => $package_url,
-    dest_folder => $package_target_dir,
-    creates => $sources,
-    uncompress => 'tar.gz'
+
+  include wget
```

Comment:
Thats not what I am asking :) you already told me why you use the wget module. The question was "Why do you need that line here? What's the purpose of `include wget`. Do you need that line?". IMHO the module should work without explicitly including wget in the manifest.